docs/design/querying.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Querying
+sidebar_position: 3
+---
+
+Quickwit is built on three pillars:
```

Comment:
same as architecture.md?

Reply:
sorry, I made some git mv and copy this one... I'm cleaning this now.